src/sentry/eventstore/models.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+from django.utils import timezone
+
+from sentry.models import EventCommon, EventDict
+from sentry.db.models import NodeData
+
+
+def ref_func(x):
+    return x.project_id or x.project.id
+
+
+class Event(EventCommon):
```

Comment:
So this is the new Event abstraction to replace the postgres model, isn't it ?
Should SnubaEvent extend this now ?

Reply:
Yes. My plan (after this gets merged and Django event is removed) is not for SnubaEvent to extend this, but for this to replace Snuba event. All the Eventstore methods would return instances of this model and SnubaEvent would be removed.